tests/unit/pages/create-or-edit-lesson/CreateOrEditLesson.spec.jsx

```diff
@@ -15,4 +22,19 @@ describe('CreateOrEditLesson', () => {
     expect(titleInput).toBeInTheDocument()
     expect(descriptionInput).toBeInTheDocument()
   })
+
+  it('should add attachments', () => {
+    const addedAttachment = screen.getByText('lesson.labels.attachments')
+    expect(addedAttachment).toBeInTheDocument()
+
+    fireEvent.click(screen.getByText('lesson.labels.attachments'))
```

Comment:
I guess, you should check smth what changed after click

Reply:
> I guess, you should check smth what changed after click

Added expect for title after click